.github/workflows/on_build_do_publish_image.yml

```diff
@@ -0,0 +1,75 @@
+name: On Build Do Publish Image
+on:
+   workflow_run:
+      workflows: [On PR Open or Push Do Build]
+      types: [completed]
+env:
+   IMAGE_NAME: ${{ vars.QUAY_IMAGE_NAME  || "quay.io/infinispan-test/server" }}
+
+concurrency:
+  # Only cancel jobs for PR updates
+  group: ci-publish-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  get-info:
+    name: "Get information about the source run"
+    runs-on: ubuntu-latest
+    outputs:
+      sourceHeadRepo: ${{ steps.workflow-run-info.outputs.sourceHeadRepo }}
+      sourceHeadBranch: ${{ steps.workflow-run-info.outputs.sourceHeadBranch }}
+      sourceHeadSha: ${{ steps.workflow-run-info.outputs.sourceHeadSha }}
+      mergeCommitSha: ${{ steps.workflow-run-info.outputs.mergeCommitSha }}
+      targetCommitSha: ${{ steps.workflow-run-info.outputs.targetCommitSha }}
+      pullRequestNumber: ${{ steps.workflow-run-info.outputs.pullRequestNumber }}
+      pullRequestLabels: ${{ steps.workflow-run-info.outputs.pullRequestLabels }}
+      targetBranch: ${{ steps.workflow-run-info.outputs.targetBranch }}
+      sourceEvent: ${{ steps.workflow-run-info.outputs.sourceEvent }}
+    steps:
+      - name: "Get information about the current run"
+        uses: potiuk/get-workflow-origin@v1_1
```

Comment:
This action has been archived since 2023, I don't think it's a good idea to rely on a dead project.

Reply:
yeah, this can be converted in API calls
I don't know why they decided to archive it, I find it pretty useful